docs/data-sources/account.md

```diff
@@ -20,6 +20,6 @@ There are no arguments available for this data source.
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The account ID.
+* `id` - The data source ID of the account.
```

Comment:
If the data source is singular， the `id` is the ID of corresponding resource. so I think it's unnecessary to update the description. Please revert others and replace *resource* with *data source* will be fine.
```suggestion
* `id` - The account ID.
```